lib/app.js

```diff
@@ -98,6 +98,14 @@ const MAPPINGS = {
   },
   currency: ['ds:8', 0, 2, 0, 0, 0, 1, 0, 1],
   priceText: ['ds:8', 0, 2, 0, 0, 0, 1, 0, 2],
+  offersIAP: {
+    path: ['ds:3', 0, 12, 12, 0],
+    fun: (val) => !!val
+  },
+  containsAds: {
```

Comment:
I think this is the same as `adSupported` which I already added (although the paths differ I think conceptually it's the same). I think `containsAds` is a better name, but the previous version already uses `adSupported` so I think it's best to keep it.

Reply:
Ok, I've just removed the `containsAds` then